tests/wait-active/wait_test.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("Elemental booting an expandable disk image", func() {
 	Context("Wait until system is expanded", func() {
 		It("eventually is active", func() {
 			Eventually(func() string {
-				out, _ := s.Command("cat /run/cos/active_mode")
+				out, _ := s.Command("cat /run/elemental/active_mode")
```

Comment:
I see upgrade test failing, I guess the image upgrading from is not having this path yet... probably we can add here a fallback to check both paths? or update image we are upgrading from.